draft/2022-11-23-this-week-in-rust.md

```diff
@@ -232,12 +218,16 @@ Rusty Events between 2022-11-23 - 2022-12-21 🦀
     * [**Rust Munich 2022 / 3 - hybrid**](https://www.meetup.com/rust-munich/events/289065390/)
 * 2022-12-01 | Virtual (Charlottesville, VA, US) | [Charlottesville Rust Meetup](https://www.meetup.com/charlottesville-rust-meetup/)
     * [**Exploring USB with Rust**](https://www.meetup.com/charlottesville-rust-meetup/events/289563986/)
+* 2022-12-01 | Virtual (Lehi, UT, US) | [Utah Rust](https://www.meetup.com/utah-rust/)
+    * [**Beginner Projects and Shop Talk with Food!**](https://www.meetup.com/utah-rust/events/289899804/)
```

Comment:
Consider adding for 12/1: https://www.meetup.com/microsoft-reactor-redmond/events/289830539/

Reply:
Added!